mev_inspect/aave_liquidations.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+from mev_inspect.schemas.classified_traces import (
+    Classification,
+    ClassifiedTrace,
+    # CallTrace,
+    DecodedCallTrace,
+)
+from mev_inspect.schemas.liquidations import Liquidation
+
+
+# Inspect list of classified traces and identify liquidation
```

Comment:
in python this sort of comment should be a docstring - example: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
Done